Packs/Active_Directory_Query/Integrations/Active_Directory_Query/Active_Directory_Query.py

```diff
@@ -660,6 +664,21 @@ def search_users(default_base_dn, page_size):
     demisto.results(demisto_entry)
 
 
+def check_if_empty_args(args: dict):
```

Comment:
What happens if this is set to true and some arguments were given? is it not valid? it is all or nothing?

Reply:
Yes, they will need to remove them, or uncheck the parameter box

Reply:
@MLainer1 Let's improve the naming of the function and the description so it will be oriented to the get-user command